docs/src/pages/components/geo/index.page.mdx

```diff
@@ -6,23 +6,76 @@ description: Amplify UI Geo provides UI components for maps and location search
 import { Example } from '@/components/Example';
 import { Feature } from '@/components/Feature';
 import { Fragment } from '@/components/Fragment';
-import { XStateInspector } from '@/components/XStateInspector';
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
 
 Amplify UI Geo provides UI components for maps and location search for popular front end frameworks, powered by
```

Comment:
super nit: I'm used to seeing "frontend" or "front-end"

Reply:
I'll change to front-end